packages/runtime/src/helpers/flags.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * If this flag is enabled, we generate individual Lambda functions for API Routes.
+ * They're packed together in 50mb chunks to avoid hitting the Lambda size limit.
+ *
+ * To prevent bundling times from rising,
+ * we use the "none" bundling strategy where we fully rely on Next.js' `.nft.json` files.
+ * This should to a significant speedup, but is still experimental.
+ *
+ * If disabled, we bundle all API Routes into a single function.
+ * This is can lead to large bundle sizes.
+ *
+ * Enabled by default. Can be disabled by passing NEXT_SPLIT_API_ROUTES=false.
+ */
+
+export const splitApiRoutes = (featureFlags: Record<string, unknown>): boolean => {
+  if (process.env.NEXT_SPLIT_API_ROUTES) {
+    return process.env.NEXT_SPLIT_API_ROUTES === 'true'
```

Comment:
We use the `destr` package helper in other places for env vars. This way also '1', true, 1, etc. is valid

```suggestion
return destr(process.env.NEXT_SPLIT_API_ROUTES)
```

Reply:
good idea, done in [f9f726f](https://github.com/netlify/next-runtime/pull/2058/commits/f9f726f5f83df1776a765b0ecf4ab6647b83ee43).